packages/wallet-connectors/CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.4.0] - 2023-11-09
+
+### Changed
+
+-   (breaking) Make compatible with `@concordium/web-sdk@^7`, add this as peer dependency.
+
+## [0.3.2] - 2023-08-17
 
 ### Changed
 
```

Comment:
```suggestion
```

something went wrong here

Reply:
You mean the breaking change is in the wrong place, I guess? I'll move it.

Reply:
No I mean the added "0.3.2" is wrong. It's an old release. It's fixed by my literal suggestion.

Reply:
Pushed a fix for this.